hash/m20.xml

```diff
@@ -470,4 +475,630 @@ license:CC0-1.0
+            <dataarea name="flop" size="286720">
+                <rom name="startrek_de.img" size="286720" crc="128ae5a7" sha1="63e9dd3f8308796c4bb02e0ba26a375188f3ed44"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="startrek_en">
+        <description>Star Trek (EN)</description>
+        <year>198?</year>
+        <publisher>None</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="startrek_en.img" size="286720" crc="2e3af580" sha1="82d72a48b9c4877092241277d29a463b861ff1e6"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="tesoro">
+        <description>Caccia al Tesoro contro il computer</description>
+        <year>1983</year>
+        <publisher>E.Petrelli</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="tesoro.img" size="286720" crc="a9c257fe" sha1="7935fcc1e0e8b417ec7954d0eeed59cc1cd8a5cc"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="topodrago">
+        <description>Il Gioco del Topo e del drago</description>
+        <year>1983</year>
+        <publisher>E.Petrelli</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="topodrago.img" size="286720" crc="b1a6cde5" sha1="522d8abf8c953502678b10f99f9b607d8e8c2c66"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="twodays1">
+        <description>Two Days to the Race (EN) - part 1</description>
+        <year>2019</year>
+        <publisher>D.Bucci</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="twodays1.img" size="286720" crc="f24ff7e0" sha1="4f6201ae7c6600fbbb44d5354d24817eb5b0ddfa"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="twodays2">
+        <description>Two Days to the Race (EN) - part 2</description>
+        <year>2019</year>
+        <publisher>D.Bucci</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="twodays2.img" size="286720" crc="d4662856" sha1="7a2b25408e69fb7acf59421d0f400d1bd00181bf"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="twodays3">
+        <description>Two Days to the Race (EN) - part 3</description>
+        <year>2019</year>
+        <publisher>D.Bucci</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="twodays3.img" size="286720" crc="1787191e" sha1="8755d37d3cc8a901098d33c3241d8af2a613f7f1"/>
+            </dataarea>
+        </part>
+    </software>
```

Comment:
Are these actually separate products, or just different disks from one package?  In the latter case, they should be parts of the same item.

Reply:
It's one package. What are the implications of the part-name field? The software list guideline says that it can be used as a command line argument. However all of these parts need to be loaded in "-flop1".

Reply:
The -flop1 command line option and the part name are not linked they just both happen to contain 'flop1'. You can list them as '<part name="flop1"...', '<part name="flop2"...', etc. From the command line they can be loaded separately using something like '-flop1 twodays:flop3'

Reply:
I have added the parent floppy, which is a single product, instead of the individual games, which were clones.